specification/sbi_cove.adoc

```diff
@@ -3,18 +3,40 @@
 [[sbi_tee]]
 == Confidential VM Extension (CoVE) SBI extension proposal
 This section describes the normative Confidential VM Extension(CoVE) SBI
-extension. This specification introduces three new extensions:
+extension. This specification introduces four new extensions:
 
+* CoVE Enumeration Extension (EXT_COVE)
 * CoVE Host Extension (EXT_COVH)
 * CoVE Interrupt Extension (EXT_COVI)
 * CoVE Guest Extension (EXT_COVG)
 
 === CoVE FID namespaces
 
-External specifications (e.g. the CoVE-IO) may need to extend one of the three
+Implementations may support more than one confidential supervisor domains. In
+order to support that scenario, as shown in <<cove-fid>>, the most significant 6
+bits of the SBI Function Id for the CoVE Extension may be set by the host to
+specifies the target Supervisor Domain Idenfier/TSM for the CoVH or COVI
+function invoked. For guest-invoked COVG functions, the RDSM must specify the
+Supervisor Domain Identifier for the callee. `a6` bits [16..25] are reserved for
+future use. `a6` bits [0..15] are used by the host to specify the CoVE functions
+as allocated in <<table_cove_fid_namespaces>>.
```

Comment:
Using a register to carry the FID is clean imho, especially from a COVG perspective. But would reserving e.g. a5 for this be preferred to overloading a6 (and somehow not fully following the SBI spec)? I'll let @atishp04 chime in here.

Reply:
I think keeping the SDID with FID seems natural - unless you think 64K space for FID may be limited.

Reply:
One of the guest function (e.g sbi_covg_get_evidence) already uses a5. The SBI calling convention doesn't specifically say something FID convention. So I am okay with overriding it as we don't have to update every function for this. 
At this point, I don't how common this scenario will be where you want to switch supervisor domains at runtime.  Thus, reserving a5 for that may be an overkill.
@avpatel any thoughts about reserving upper bits of FID for SDID ?

Reply:
there are other attestation functions that will also use a5 - so would prefer to not burn a register and use the extra available space in a6 (FID).